Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -83,7 +83,6 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
       |> UIButton.lens.image(for: .selected) .~ Library.image(named: "mobile-icon",
                                                       tintColor: .ksr_green_700,
                                                       inBundle: Bundle.framework)
-      |> UIButton.lens.accessibilityLabel %~ { _ in Strings.Push_notifications() }
 
```

Comment:
Noticed that there was previously an empty line here, can we possibly remove it?